config.env

```diff
@@ -0,0 +1 @@
+ATLAS_URI=mongodb+srv://Hililevitan:Hililevitan7cluster1.mw4wkng.mongodb.net/test
```

Comment:
this is secret! never add secrets to git.
add `config.env` to gitignore!

Reply:
done